hamocc/mo_dipowa.F90

```diff
@@ -143,7 +144,7 @@ subroutine dipowa(kpie,kpje,kpke,omask,lspin)
           l = ks-k
           do i = 1,kpie
             if (omask(i,j) > 0.5) then
-              powtra(i,j,l,iv) = ( sedb1(i,l,iv)                               &
+              powtra(i,j,l,iv) = ( sedb1(i,l,iv)                                                   &
```

Comment:
doesn't fit in one line? - but maybe its just a bit over the line - not sure, if we should see this a bit sloppy...

Reply:
No, doesn't fit. Since it was like this before, I left it.